modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/image/ImageDDMFormFieldValueRendererTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.dynamic.data.mapping.form.field.type.internal.image;
+
+import com.liferay.dynamic.data.mapping.storage.DDMFormFieldValue;
+import com.liferay.dynamic.data.mapping.test.util.DDMFormValuesTestUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Nathaly Gomes
+ */
+public class ImageDDMFormFieldValueRendererTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Test
+	public void testRender() {
+		DDMFormFieldValue ddmFormFieldValue =
+			DDMFormValuesTestUtil.createDDMFormFieldValue(
+				StringUtil.randomString(), null);
+
+		ImageDDMFormFieldValueRenderer imageDDMFormFieldValueRenderer =
+			new ImageDDMFormFieldValueRenderer();
+
+		Assert.assertEquals(
+			StringPool.BLANK,
+			imageDDMFormFieldValueRenderer.render(
+				ddmFormFieldValue, LocaleUtil.US));
```

Comment:
@vngoliveira can you also inline the `ddmFormFieldValue`? If not, please reorder:

```
ImageDDMFormFieldValueRenderer imageDDMFormFieldValueRenderer = ...

DDMFormFieldValue ddmFormFieldValue = ....
```